### DISCUSSION TITLE
uStepper S32 simultaneous motor control help needed please!

### DISCUSSION THREAD

**Discussion by JenRob2024** [grblHAL/core#424] [Q&A]
Hi, I'm a bit out of my depth, but have 2 x uStepper S32 controlled motors (NEMA23) with uStepper Wifi GUI Shields attached, all set up and working individually. I've realised that the GUI doesn't support multiple motors, but I would like to control them simultaneously to drive 2 x leadscrews (and am hoping the encoder allows accurate positioning of each).
Can anyone explain how I might achieve this best please?

**Comment by terjeio** [comment]
If they can be controlled with step/dir signals it is just a matter of connecting them.
Or do you want to do something [like this](https://github.com/grblHAL/core/discussions/405)?
